src/components/KeyboardAvoidingView/index.ios.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * The KeyboardAvoidingView is only used on ios
+ */
+import React from 'react';
+import {KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    children: PropTypes.node,
+};
+const defaultProps = {
+    children: null,
```

Comment:
Should the `behavior` prop get a default here? I seem to recall the RN docs saying that omitting that prop entirely could have unknown side-effects (or something like that).

Reply:
+1 I see this in the docs:

>Android and iOS both interact with this prop differently. On both iOS and Android, setting behavior is recommended.

Reply:
The ScreenWrapper prop file has `keyboardAvoidingViewBehavior: 'padding',` as default, but I think we should make it optional there and use a default behavior here instead.